### ISSUE TITLE
Show Calendar In Each Column for Day View

### ISSUE THREAD

**Issue by sambadabudiarga** [nhn/tui.calendar#783]
## Version
1.12.14

## Development Environment
Browser: Firefox and Chromium
OS: Linux Manjaro

## Current Behavior
Day View currently shows all calendars into one big column, based on how the time overlapped.

## Expected Behavior
Another option for Day View which allows each calendar in their own column.
This is to make moving events between calendars easier.
I don't suggest it to be available for Week View or even Month View as it is not really usable.

**Comment by jungeun-cho** [comment]
For example, do you want to display the columns of the daily view as [Mon, Wed, Fri] or [Mon, Tue]??

**Comment by sambadabudiarga** [comment]
No. For example, each calendar I have is an operator that should do a job.
Here is a sample of a calendar I have created.
![image](https://user-images.githubusercontent.com/20062600/116367356-db40ee80-a839-11eb-945e-6538fd36eb88.png)

From there, you can see that there are multiple operators, which I considered each one as a calendar.
They are, "Janvik", "Operator 8" and "Operator 11".
I would like each of those to be in their own separate column.
So that Janvik could be in the 1st column, Operator 8 in the 2nd column, and Operator 11 in the last column.

Please let me know if I am not being clear enough to you.

**Comment by sambadabudiarga** [comment]
Hi,

I am sorry if I am demanding here, but is there anything I can help with? Maybe another information you would need from me regarding this?

Thank you.

**Comment by sambadabudiarga** [comment]
The work around I made currently is by using the html of the detail body.
Where we can have a button over there, that when being clicked, it will
shows a popup form with option of the staffs/operators.

But really would like if this feature can be implemented natively.

Sent from mobile device.

Sambada Budiarga
Software Developer
e: ***@***.***

**Comment by adhrinae** [comment]
@sambadabudiarga 

> The work around I made currently is by using the html of the detail body.
> Where we can have a button over there, that when being clicked, it will
> shows a popup form with option of the staffs/operators.

That's interesting. I'd like to see any screenshot or example of your idea.

**Comment by Mr-Rayman** [comment]
I definitely would second that request of managing multiple resources in separate columns in day view.

This is also emphasized in other threads and requests:

https://github.com/nhn/tui.calendar/issues/430

https://github.com/nhn/tui.calendar/issues/438

https://github.com/nhn/tui.calendar/issues/567

Thanks!

**Comment by adhrinae** [comment]
@Mr-Rayman I understand that there are many requests so I'll keep in mind those requests.

**Comment by goltra** [comment]
Hello everybody, I'd like to know how goes this feature. @adhrinae do you have an estimated date? I was thinking load multiple instance but after reading this thread I  think is not a good idea.
thanks

**Comment by adhrinae** [comment]
@goltra 
There's no planned release date yet, but we are working on the next major version which can accept various requests easier.

You can try setting the multiple instances as long as you don't customize the `template` option separately by instance.

**Comment by goltra** [comment]
> @goltra There's no planned release date yet, but we are working on the next major version which can accept various requests easier.
> 
> You can try setting the multiple instances as long as you don't customize the `template` option separately by instance.

You mean set template in a variable and assign it to all calendar instances

**Comment by adhrinae** [comment]
@goltra 

It's a known bug as #227.
The only `template` property set from the last instance will be applied to the whole instance. Unless you isolate instances to make reference to different `window` objects, such as `iframe`.

**Comment by sambadabudiarga** [comment]
As promised, I've created a workaround for this feature by using a custom popup of an event like this,

![event popup](https://user-images.githubusercontent.com/20062600/157165600-a90d3998-a956-40c2-a374-2e5264fc4b34.png)

So when that "Edit" button is clicked, it will basically trigger another popup form, where we can change the calendar for that specific event.

**Comment by adhrinae** [comment]
@sambadabudiarga 

I wonder how the custom popup related to the feature request you mentioned the first time.
At initial, you needed that each of the operators to have separated columns. But the screenshot only shows the way to edit a specific event.

**Comment by vineyrawat** [comment]
Hi @adhrinae , I also need the same column feature as above for the day view where I can see the resource like Room1, Room2, and Room3, could you please guide me on how can I do it or if you've implemented it? Thanks in advance
